matlab/test/arrow/tabular/tRecordBatch.m

```diff
@@ -28,11 +28,15 @@ function Basic(tc)
         function SupportedTypes(tc)
             % Create a table all supported MATLAB types.
             import arrow.internal.test.tabular.createTableWithSupportedTypes
+            import arrow.type.traits.traits
 
             TOriginal = createTableWithSupportedTypes();
             arrowRecordBatch = arrow.recordBatch(TOriginal);
             expectedColumnNames = string(TOriginal.Properties.VariableNames);
-            tc.verifyRecordBatch(arrowRecordBatch, expectedColumnNames, TOriginal);
+
+            expectedArrayClasses = varfun(@(var) traits(string(class(var))).ArrayClassName, ...
```

Comment:
Could you add a comment like the following here: "For each variable in the input MATLAB table, look up the corresponding Arrow Array type using type traits."

Reply:
done!